.travis.yml

```diff
@@ -63,27 +62,31 @@ notifications:
     skip_join: true
 
 before_deploy:
- - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
+ - if [[ "${TRAVIS_OS_NAME}" == "linux" ]]; then
      wget https://mirrors.edge.kernel.org/ubuntu/pool/universe/n/nsis/nsis-common_3.04-1_all.deb;
      wget https://mirrors.edge.kernel.org/ubuntu/pool/universe/n/nsis/nsis_3.04-1_amd64.deb;
      sudo dpkg -i nsis-common_3.04-1_all.deb;
      sudo dpkg -i nsis_3.04-1_amd64.deb;
-     cd packaging && ./update-wiki.sh ${TRAVIS_TAG} && cd ..;
-   fi;
- - export PATH=$PATH:$HOME/usr/bin
+     echo ${TRAVIS_REPO_SLUG};
+     if [[ "${TRAVIS_REPO_SLUG}" == "OpenRA/OpenRA" ]]; then
```

Comment:
The rest of these changes disable things that I otherwise need to patch out when building devtests. It is finally possible to build and deploy builds to forks without changing the travis config.